hw/top_earlgrey/rdc/rdc_waivers.tcl

```diff
@@ -130,3 +130,42 @@ set_rule_status -rule E_RST_METASTABILITY -status Waived \
   -expression {(SourceReset=="POR_N") && \
     (ResetFlop=~"*.u_clkmgr_aon.*usb_peri_sw_en_sync*") && \
     (MetaStableFlop=~"*.u_clk_usb_peri_cg.*")}
+
+# EN sync to EN Latch in CG
+# (io_div4|io_div2|io|usb)_root_ctrl
+create_view_criteria -name PorNRootCtrlCg -rule E_RST_METASTABILITY \
+  -criteria {(SourceReset=="POR_N") && \
+    (ResetFlop=~"*.u_clkmgr_aon.*root_ctrl*") && \
+    (MetaStableFlop=~"*.u_*_root_ctrl.u_cg*")}
+set_rule_status -rule E_RST_METASTABILITY -status Waived \
+  -expression {(SourceReset=="POR_N") && \
+    (ResetFlop=~"*.u_clkmgr_aon.u_main_root_ctrl.u_cg*") && \
+    (MetaStableFlop=~"*.u_clkmgr_aon.u_main_root_ctrl.u_cg*")}
+set_rule_status -rule E_RST_METASTABILITY -status Waived \
+  -expression {(SourceReset=="POR_N") && \
+    (ResetFlop=~"*.u_clkmgr_aon.u_io_div4_root_ctrl.u_cg*") && \
+    (MetaStableFlop=~"*.u_clkmgr_aon.u_io_div4_root_ctrl.u_cg*")}
+set_rule_status -rule E_RST_METASTABILITY -status Waived \
+  -expression {(SourceReset=="POR_N") && \
+    (ResetFlop=~"*.u_clkmgr_aon.u_io_div2_root_ctrl.u_cg*") && \
+    (MetaStableFlop=~"*.u_clkmgr_aon.u_io_div2_root_ctrl.u_cg*")}
+set_rule_status -rule E_RST_METASTABILITY -status Waived \
+  -expression {(SourceReset=="POR_N") && \
+    (ResetFlop=~"*.u_clkmgr_aon.u_io_root_ctrl.u_cg*") && \
+    (MetaStableFlop=~"*.u_clkmgr_aon.u_io_root_ctrl.u_cg*")}
+set_rule_status -rule E_RST_METASTABILITY -status Waived \
+  -expression {(SourceReset=="POR_N") && \
+    (ResetFlop=~"*.u_clkmgr_aon.u_usb_root_ctrl.u_cg*") && \
+    (MetaStableFlop=~"*.u_clkmgr_aon.u_usb_root_ctrl.u_cg*")}
+
+# Waiver: PowerDown to AST Regulator
+# No need of verifying AST behavioral model
```

Comment:
do we need to confirm nuvoton has synchronizer cells here? or whether they even need them? sorry maybe we already did this and i just can't remember.

Reply:
Good point! The line is POR_N and deep sleep. I assumed the device is not in deep sleep when POR_N is asserted. 

@arnonsha The `main_pd_ni` in the regulator will be dropped asynchronously when POR_N is asserted. Does regulator block have synchronizer?

Reply:
Unfortunately we don't. We didn't expect it to be asynchronous. Spec defines it as sync to aon clock.
I run simulation and it looks like we have another problem:
When in deep sleep and por_n is asserted both main_pd_n and power_clamp (iso en) are de-asserted at the same time (asynchronously). AON is on, but there's still no MAIN power. However clamping is not enabled any more which leads to Xs going from main to AON !!

@zi-v 
@meisnere

Reply:
@arnonsha / @zi-v.. i think there are probably a few things.

`POR_N` is not directly used on our side, it is first sequenced through `ast`. So I think this means we don't "have" to synchronize the `main_pd_ni` input as long as you control the `vcaon_pok` to be on `aon clock` also.  What do you think?

Regarding the reset value, that is a good point. If you recall, quite a long time ago, we made the explicit decision to make the "default state" on (see #5170).  I think @zi-v point at the time is that on POR, the regulator is already ready, so we do not need to reset to off and turn it on again.  Is that still true now? In which case it seems like there should not be a problem to default to these values right? If not, I think we can change back as well.
@jeoongp @matutem